pkg/storage/tsdb/block/meta.go

```diff
@@ -44,7 +45,8 @@ const (
 type Meta struct {
 	tsdb.BlockMeta
 
-	Thanos ThanosMeta `json:"thanos"`
+	Thanos       ThanosMeta `json:"thanos"`
+	LastModified *time.Time `json:"-"`
```

Comment:
`Meta` struct describes meta.json object on long-term storage or in local cache, and I don't think we should be adding more fields to this struct that we don't actually store.